docs/index.rst

```diff
@@ -10,5 +10,5 @@ Documentation content:
    :maxdepth: 1
 
    User guide <userguide/index>
-   Development guide <development>
+   Development guide <development/index>
```

Comment:
I'm pretty sure that moving the dev guide into a subfolder will break the inclusion of the fragments guide that resides under `changelog.d/`. So this may need to be corrected.